hawc/apps/materialized/managers.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+
+class FinalRiskOfBiasScoreQuerySet(models.QuerySet):
+    def prioritized_scores(self):
```

Comment:
i dont think this will work in the way you intend.  The reason is for a given set of RiskOfBiasScore, we'll likely want 10 defaults, 1 animal group override, and 2 endpoint overrides, one score for each metric id. Suggest removing for now.

Reply:
I went ahead and removed it, but I left the boilerplate in place for custom manager functions / queryset functions down the line